src/components/smartSearch/SmartSearchDialog/QueryOverview.tsx

```diff
@@ -16,10 +17,23 @@ import DisplaySurveyOption from '../filters/SurveyOption/DisplaySurveyOption';
 import DisplaySurveyResponse from '../filters/SurveyResponse/DisplaySurveyResponse';
 import DisplaySurveySubmission from '../filters/SurveySubmission/DisplaySurveySubmission';
 import DisplayUser from '../filters/User/DisplayUser';
-import { AnyFilterConfig, CallHistoryFilterConfig, CampaignParticipationConfig, FILTER_TYPE,
-    MostActiveFilterConfig, PersonDataFilterConfig, PersonTagsFilterConfig, RandomFilterConfig,
-    SmartSearchFilterWithId, SubQueryFilterConfig, SurveyOptionFilterConfig,
-    SurveyResponseFilterConfig, SurveySubmissionFilterConfig, UserFilterConfig } from 'types/smartSearch';
+import {
+    AnyFilterConfig,
+    CallHistoryFilterConfig,
+    CampaignParticipationConfig,
+    FILTER_TYPE,
+    MostActiveFilterConfig,
+    PersonDataFilterConfig,
+    PersonFieldFilterConfig,
+    PersonTagsFilterConfig,
+    RandomFilterConfig,
+    SmartSearchFilterWithId,
+    SubQueryFilterConfig,
+    SurveyOptionFilterConfig,
+    SurveyResponseFilterConfig,
+    SurveySubmissionFilterConfig,
+    UserFilterConfig,
```

Comment:
I've been wanting to do this for a long time (mostly because I like it, but also because it helps avoid merge conflicts) and now I did it to fix the merge conflict that arose.